src/scripts/main.js

```diff
@@ -1,3 +1,32 @@
 'use strict';
 
-// write your code here
+const promise1 = new Promise((resolve) => {
+  document.querySelector('.logo').addEventListener('click', () => {
+    resolve();
```

Comment:
```suggestion
    resolve('Promise was resolved!');
```
and get it as an argument in the `handleSuccess` function to use as text content